FirebaseDynamicLinks/README.md

```diff
@@ -0,0 +1,25 @@
+# Firebase Dynamic Links SDK for iOS
+
+Firebase Dynamic Links are links that work the way you want, on multiple platforms, and whether or not your app is already installed.
+For more info : https://firebase.google.com/products/dynamic-links
+
+Please visit [our developer site]
+(https://firebase.google.com/docs/dynamic-links/) for integration instructions,
+documentations, support information, and terms of service.
+
+## Managing the Pasteboard
+
+This release of the FDL SDK use a plist property (FirebaseDeepLinkPasteboardRetrievalEnabled) that a developer can set to enable/disable the use of iOS pasteboard by the SDK.
```

Comment:
nit: replace `use` with `adds`

Reply:
it was changed from adds to use based on an early comment. Please confirm if this change back is must.

Reply:
Hm. Maybe leave it as `use` and replace `This release of the FDL SDK` with `Firebase Dynamic Links 4.2.0 and higher`?

I'm also ok with just ignoring my nit comment above.

Reply:
+1 for replacing the "This release" with "Version 4.2.0 and higher"

Reply:
Done